ugc/serdes.hpp

```diff
@@ -50,6 +51,11 @@ class Serializer
     WriteToSink(m_sink, index);
   }
 
+  void VisitLangs(std::vector<uint8_t> const & indexes, char const * /* name */ = nullptr)
```

Comment:
`#include <vector>`?

Reply:
done

Reply:
Why do we need this method at all?

Reply:
Because we use it. Actually, I didn't get the question.

Reply:
I mean, it looks like you may remove this mehod and replace visitLangs(m_indexes, "...") by visitor(m_index), right?

Reply:
No, I can't, because we need to transform ```uint8_t``` to ```char*``` while we serialize into json.